README.md

```diff
@@ -1,109 +1,148 @@
 
-## Installation Instructions
-Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+## Step 2: Install the package
+Include the following Marketo package version in your `packages.yml` file.
 
-Include in your `packages.yml`
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 
-```yaml
+```yml
 packages:
   - package: fivetran/marketo
-    version: [">=0.7.0", "<0.8.0"]
+    version: [">=0.8.0", "<0.9.0"]
 ```
 
-## Configuration
-By default, this package will look for your Marketo data in the `marketo` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Marketo data is , please add the following configuration to your `dbt_project.yml` file:
+## Step 3: Define database and schema variables
+By default, this package runs using your destination and the `marketo` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Marketo data is (for example, if your Marketo schema is named `marketo_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
+vars:
+  marketo_database: your_database_name
+  marketo_schema: your_schema_name 
+```
 
-...
-config-version: 2
+For additional configurations for the source models, please visit the [Marketo source package](https://github.com/fivetran/dbt_marketo_source).
 
+## Step 4: Enabling/Disabling Models
+This package takes into consideration tables that may not be synced due to slowness caused by the Marketo API.  By default the `campaign` and `program` related-models are disabled.  If you sync these tables, enable the modeling done by adding the following to your `dbt_project.yml` file:
+
+```yml
 vars:
-  marketo_source:
-    marketo_database: your_database_name
-    marketo_schema: your_schema_name 
+    marketo__enable_campaigns:   true      # Enable if Fivetran is syncing the campaign table
+    marketo__enable_programs:    true      # Enable if Fivetran is syncing the program table
 ```
 
-For additional configurations for the source models, please visit the [Marketo source package](https://github.com/fivetran/dbt_marketo_source).
+Alternatively, you may need to disable certain models. The below models can be disabled by adding them to your `dbt_project.yml` file:
+```yml
+vars:
+    marketo__activity_delete_lead_enabled:  false     # Disable if you do not have the activity_delete_lead table 
+```
+## (Optional) Step 5: Additional configurations
```

Comment:
Sorry I missed this, can you add the collapsable portion for step 5

Reply:
This was my bad too. I took it out while editing and forgot to put it back. Added!